app/src/main/java/com/example/coinage/ParseApplication.java

```diff
@@ -0,0 +1,20 @@
+package com.example.coinage;
+
+import android.app.Application;
+
+import com.parse.Parse;
+import com.parse.ParseObject;
+
+public class ParseApplication extends Application {
+    // initializes parse sdk as soon as the application is created
+    @Override
+    public void onCreate() {
+        super.onCreate();
+
+        Parse.initialize(new Parse.Configuration.Builder(this)
+                .applicationId(getString(R.string.back4app_app_id))
```

Comment:
this may work.. but I was thinking around these lines:
    public static final String BACK4APP_APP_ID = "aaaabbbbbbcccccxxxxxyyyyzzzz";

Reply:
I see, is it generally better practice to use constants in the files instead of the strings file? What if the same constants were being used in a different file as well?